webapp/platform/components/src/generic_modal/__snapshots__/generic_modal.test.tsx.snap

```diff
@@ -0,0 +1,143 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`GenericModal should match snapshot for base case 1`] = `
```

Comment:
while running my tests locally it had to update this snapshot since it was outdated.